### PR TITLE
Add section for "Disk Utility"

### PR DESCRIPTION
**Pull request by issmirnov**
If this is accepted, I can also send one for `defaults write com.apple.DiskUtility advanced-image-options -bool true`.

### CODE REVIEW

File: defaults.yml [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
+        param:
+          type: bool
+        examples:
+          - value: false
+            default: true
+            text: Don't show additional options in disk utility
+          - value: true
+            text: Show additional options in disk utility
+        versions: [Big Sur, Catalina, Mojave]
+        after: killall Disk Utility
```
Comment (yannbertrand) [inline defaults.yml:24]:
```suggestion
        after: killall "Disk Utility"
```

File: defaults.yml [hunk 2]
```diff
@@ -2,6 +2,27 @@
 
 categories:
 
+  - folder: disk-utility
+    name: Disk Utility
+    description:
+      The Disk Utility application is used to manage disks on mac.
+      # TODO(someone): add image.
```
Comment (yannbertrand) [inline defaults.yml:9]:
```suggestion
```

**Review by yannbertrand — COMMENTED** [review]
Thanks! Could you show me what the menu looks like? I can't find it

**Comment by issmirnov** [comment]
Hey,

Apologies about delayed response, work has been wild.

I looked for the menu, but it seems that with Catalina+ this is no longer the correct command to view advanced options. I'm closing this PR, since folks don't care about commands for 10.10 and under :)

**Pull request closed without merging**